inspire/base/templates/accounts/settings/helpers.html

```diff
@@ -0,0 +1,58 @@
+{#
+## This file is part of INSPIRE.
+## Copyright (C) 2015 CERN.
+##
+## INSPIRE is free software; you can redistribute it and/or
+## modify it under the terms of the GNU General Public License as
+## published by the Free Software Foundation; either version 2 of the
+## License, or (at your option) any later version.
+##
+## INSPIRE is distributed in the hope that it will be useful, but
+## WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+## General Public License for more details.
+##
+## You should have received a copy of the GNU General Public License
+## along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
+## 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+#}
+
+ {%- extends "accounts/settings/helpers_base.html" -%}
```

Comment:
what is the difference between this template and the original one? https://github.com/inveniosoftware/invenio/blob/master/invenio/modules/accounts/templates/accounts/settings/helpers_base.html

Can we just overwrite some block instead of copying it all over?